README.md

```diff
@@ -1,15 +1,13 @@
-# Starter Project
----
+# SpatialOS Unity SDK Starter Project
 
-*Copyright (C) 2017 Improbable Worlds Limited. All rights reserved.*
 
-- *GitHub repository*: [https://github.com/spatialos/StarterProject](https://github.com/spatialos/StarterProject)
+- *GitHub repository*: [github.com/spatialos/StarterProject](https://github.com/spatialos/StarterProject)
+
 
----
 
 ## Introduction
 
-This is a SpatialOS starter project with useful core features that you can extend to build your own SpatialOS application.
+This is a [SpatialOS](https://docs.improbable.io/reference/latest/shared/concepts/spatialos) starter project to use with the [Unity SDK](https://github.com/spatialos/UnityGDK) with useful core features that you can extend to build your own SpatialOS application.
```

Comment:
I think the second link here should go to /spatialoos/UnitySDK, rather than UnityGDK?

Reply:
Yes! Got GDK on brain! Thanks.

Reply:
Done.